packages/terra-time-input/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -55,7 +55,7 @@
 
   --terra-time-input-disabled-background-color: #222a2e;
   --terra-time-input-disabled-border: 1px solid #181b1d;
-  --terra-time-input-disabled-color: #b2b5b6;
+  --terra-time-input-disabled-color: #404344;
```

Comment:
this seems shady what are we doing here?

Reply:
This was used before because terra-form-input had higher specificity. Now that time-input has higher specificity and this is being used. One of the tests was failing because the disabled color didn't match. This is to match terra-form-input: https://github.com/cerner/terra-core/blob/master/packages/terra-form-input/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss#L10

Reply:
Thanks for the explanation!